README.md

```diff
@@ -111,6 +111,9 @@ First you will want to install the following packages by executing the following
 `sudo apt-get update`
 `sudo apt-get install -y python3 python3-pip sqlite3 git unzip libmagickwand-dev`
 
+Then install the Zenora library for python by executing this command
+`pip install Zenora`
```

Comment:
Why?
Add it to the requirements via pip freeze

Reply:
Yep, this is corrected now

Its been added to requirements